BookEcommerce.Models/DAL/Repositories/AuthenticationRepository.cs

```diff
@@ -48,15 +48,15 @@ DbFactory dbFactory
 
         public async Task<IdentityResult> CreateAdmin(ApplicationUser ApplicationUser, string password)
```

Comment:
ApplicationUser => applicationUser. Btw, ApplicationUser is not clear to me when create Admin

Reply:
resolved